scripts/contentScript.js

```diff
@@ -351,8 +384,8 @@ async function FilesInPrBitbucket(response) {
 		if (!ticking) {
 			window.requestAnimationFrame(() => {
 				if (currentScrollPosition - lastKnownScrollPosition > 100) {
-					if ("relevant" in response) {
-						const encryptedFileNames = new Set(response['relevant']);
+					if ("files" in response) {
+						const encryptedFileNames = new Set(response['files']);
```

Comment:
Changed contract in new apis

Reply:
Reason?
As far as I remember, we were about to eliminate an API call here. So what is the reason behind modifying the field name in the meanwhile?

(Also, I hope you have maintained backward compatibility in the backend API)

Reply:
Oh, nevermind. Didn't see that this function is only specific to Bitbucket and that we can moving the function from Python to NextJS.